cpanfile

```diff
@@ -17,7 +17,7 @@ on 'develop' => sub {
     requires 'Code::TidyAll';
     requires 'Perl::Critic';
     requires 'Perl::Critic::Community';
-    requires 'Perl::Tidy', '== 20230912';
+    requires 'Perl::Tidy', '== 20240202';
```

Comment:
please revert that

Reply:
But then the check will complain.
The check here is using the version from CPAN.

Reply:
First suggestion for the future: The commit message can explain that. So not just "Update perltidy version" which is obvious but why. And doesn't the CPAN install parse the version string?